### PR TITLE
Kill profiles.

### PR DESCRIPTION
**Pull request by jforberg**
It does what it says in the headline.

### CODE REVIEW

File: app/models/cafe_work.rb [hunk 1]
```diff
@@ -33,17 +33,7 @@ def send_email
...
-      if user_id == nil
-        return User.find_by(id: profile_id)
-      else
-        return user
-      end
-    else
-      return user
-    end
-  end
+  alias old_profile user
```
Comment (jforberg) [inline app/models/cafe_work.rb]: @davidwessman Please verify that this is correct. I'm not sure I understand what the code is supposed to be doing.

Comment (davidwessman) [inline app/models/cafe_work.rb]: LGTM
I do not think this code is used somewhere else, I probably had some idea to use this during the merge Profile -> User.
Perhaps keep it as an alias for a while.

Comment (jforberg) [inline app/models/cafe_work.rb]: It's used in the cafework views.

File: db/migrate/20150906182103_kill_profiles.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class KillProfiles < ActiveRecord::Migration
+  def change
+    drop_table :profiles
+    drop_table :posts_profiles
+    [ :cafe_works, :candidates, :documents, :news, :rents ].each do |t|
```
Comment (jforberg) [inline db/migrate/20150906182103_kill_profiles.rb]: Detect this

article_img

**Pull request merged**